Fabric.Identity.UnitTests/Validation/StubTokenValidator.cs

```diff
@@ -17,6 +19,11 @@ public Task<TokenValidationResult> ValidateAccessTokenAsync(string token, string
             bool validateLifetime = true)
         {
             return Task.FromResult(IdentityTokenValidationResult);
-        }
+        }
+
+        public Task<TokenValidationResult> ValidateRefreshTokenAsync(string token, Client client = null)
```

Comment:
why are parameters being passed, but not used?

Reply:
I think that an interface was updated, and in this case, where this is a StubTokenValidator just used for testing purposes, I implemented the method, but without any real content in the method.